packages/web-wallet/pages/contacts/show/index.tsx

```diff
@@ -0,0 +1,550 @@
+              //   "display": [
+              //     {
+              //       "name": "GPA"
+              //     }
+              //   ]
+              // }
+            }
+          },
+          "proof_types_supported": {
+            "jwt": {
+              "proof_signing_alg_values_supported": [
+                "ES256"
+              ]
+            }
+          },
+          "display": [
+            {
+              "name": "Driving License Credential",
+              "description": "Proof of ability to drive a car",
+              "locale": "en-US",
+              "logo": {
+                "url": "https://img.freepik.com/free-vector/bird-colorful-logo-gradient-vector_343694-1365.jpg?size=338&ext=jpg&ga=GA1.1.2008272138.1723680000&semt=ais_hybrid",
+                "alt_text": "a square logo of a university"
+              },
+              "background_color": "#feffa3",
+              "text_color": "#FFFFFF"
+            }
+          ]
+        },
+      }
+
+      if (Array.isArray(credentialsSupported)) {
+        throw Error('Only OID4VCI v13 is supported for now')
+      }
+
+      return credentialsSupported
+    } catch (error) {
+      // @ts-ignore // TODO
+      throw Error(`Error fetching credential supported. ${error.message}`)
+    }
+  }
+
+  const getCredentialBranding = async (args: getCredentialBrandingArgs): Promise<Record<string, Array<IBasicCredentialLocaleBranding>>> => {
+    const { credentialsSupported } = args
+    const credentialBranding: Record<string, Array<IBasicCredentialLocaleBranding>> = {}
+    await Promise.all(
+        Object.entries(credentialsSupported).map(async ([configId, credentialsConfigSupported]) => {
+          const localeBranding: Array<IBasicCredentialLocaleBranding> = await Promise.all(
+              (credentialsConfigSupported.display ?? []).map(
+                  async (display: CredentialsSupportedDisplay): Promise<IBasicCredentialLocaleBranding> =>
+                      await agent.ibCredentialLocaleBrandingFrom({ localeBranding: await credentialLocaleBrandingFrom(display) }),
+              ),
+          )
+
+          credentialBranding[configId] = localeBranding
+        }),
+    )
+
+    return credentialBranding
+  }
+
+  const selectCredentialLocaleBranding = (args: SelectCredentialLocaleBrandingArgs): IBasicCredentialLocaleBranding | undefined => {
+    const { locale, localeBranding } = args
+    return localeBranding?.find((branding: IBasicCredentialLocaleBranding) => locale
+        ? branding.locale?.startsWith(locale) || branding.locale === undefined
+        : branding.locale === undefined
+    )
+  }
+
+  useEffect(() => {
+    if (isLoading) {
+      return
+    }
+
+    const credentialIssuer= partyData?.data.identities.find((identity) => identity.roles.includes(CredentialRole.ISSUER))?.identifier.correlationId
+    if (!credentialIssuer) {
+      return
+    }
+
+    OpenID4VCIClient.fromCredentialIssuer({
+      credentialIssuer,
+      authorizationRequest: {
```

Comment:
Should not be needed to set. As we will only retrieve the server metadata

Reply:
removed